codice/ddf
# DDF-04372 Migrates webpack to use `MiniCssExtractPlugin`

#### What does this PR do?

Migrates webpack to use `MiniCssExtractPlugin`

#### Who is reviewing it? 

@zta6 
@gjvera 
@willwill96 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining


#### How should this be tested?

Open any production version of a UI using webpack and ensure the styles load correctly.

#### What are the relevant tickets?

For GH Issues:
Fixes: #4372


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
